src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
```suggestion
           private static final int BALLS_COUNT = 3;
    public static void main(String[] args) {
        ColorSupplier colorSupplier = new ColorSupplier();
        Lottery lottery = new Lottery(colorSupplier);
        for (int i = 0; i < BALLS_COUNT; i++) {
            System.out.println(lottery.getRandomBall());
        }
    }
```

You should create only one Lottery object for getting random ball and use for loop instead of duplication of code. Don't call toString() method on ball object because it's called "under the hood". Use BALLS_COUNT variable for count of loop iterations.